src/test/java/guitests/UITest.java

```diff
@@ -362,6 +362,29 @@ private void awaitCondition(Callable<Boolean> condition, int timeoutInSeconds) {
     }
 
     /**
+     * Performs logout from File -> Logout on HubTurbo's pView.
+     */
+    public void logout(){
+        clickMenu("File", "Logout");
+    }
+
+    /**
+     * Performs UI login on the login dialog box.
+     * @param owner
+     * @param repoName
+     * @param username
+     * @param password
```

Comment:
Side question: can you check if these 4 lines add any value when the IDE generates popup documentation for method calls?